content/id/docs/concepts/containers/container-environment-variables.md

```diff
@@ -1,59 +1,59 @@
 ---
-title: Variabel Environment Kontainer
+title: Variabel Environment Container
 content_template: templates/concept
 weight: 20
 ---
 
 {{% capture overview %}}
 
-Laman ini menjelaskan berbagai *resource* yang tersedia di dalam Kontainer pada suatu *environment*.
+Laman ini menjelaskan berbagai *resource* yang tersedia di dalam Container pada suatu *environment*.
 
 {{% /capture %}}
 
 
 {{% capture body %}}
 
-## *Environment* Kontainer
+## *Environment* Container
 
-*Environment* Kontainer pada Kubernetes menyediakan beberapa *resource* penting yang tersedia di dalam Kontainer:
+*Environment* Container pada Kubernetes menyediakan beberapa *resource* penting yang tersedia di dalam Container:
 
 * Sebuah *Filesystem*, yang merupakan kombinasi antara [image](/docs/concepts/containers/images/) dan satu atau banyak [*volumes*](/docs/concepts/storage/volumes/).
-* Informasi tentang Kontainer tersebut.
+* Informasi tentang Container tersebut.
 * Informasi tentang objek-objek lain di dalam kluster.
 
-### Informasi tentang Kontainer
+### Informasi tentang Container
 
-*Hostname* sebuah Kontainer merupakan nama dari Pod dimana Kontainer dijalankan.
+*Hostname* sebuah Container merupakan nama dari Pod dimana Container dijalankan.
 Informasi ini tersedia melalui perintah `hostname` atau panggilan (*function call*)
 [`gethostname`](http://man7.org/linux/man-pages/man2/gethostname.2.html) pada `libc`.
 
 Nama Pod dan *namespace* tersedia sebagai variabel *environment* melalui [API *downward*](/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information/).
 
-Variabel *environment* yang ditulis pengguna dalam Pod *definition* juga tersedia di dalam Kontainer,
+Variabel *environment* yang ditulis pengguna dalam Pod *definition* juga tersedia di dalam Container,
 seperti halnya variabel *environment* yang ditentukan secara statis di dalam *image* Docker.
 
 ### Informasi tentang Kluster
 
-Daftar semua *Service* yang dijalankan ketika suatu Kontainer dibuat, tersedia di dalam Kontainer tersebut sebagai variabel *environment*.
+Daftar semua *Service* yang dijalankan ketika suatu Container dibuat, tersedia di dalam Container tersebut sebagai variabel *environment*.
```

Comment:
I think we need to standardize whether first letter ini container word need to capitalize or not

Reply:
@girikuncoro what do you think?

Reply:
cc @phanama as well